src/components/molecules/TableGrid/TableGrid.tsx

```diff
@@ -1,59 +1,98 @@
 import React, { useCallback, useEffect, useMemo, useState } from "react";
+import { useVideoHuddle } from "components/attendee/VideoHuddle/useVideoHuddle";
 import { groupBy } from "lodash";
 
 import { ALLOWED_EMPTY_TABLES_NUMBER } from "settings";
 
-import { setTableSeat } from "api/venue";
+import { setTableSeat, unsetTableSeat } from "api/venue";
 
-import { Table, TableComponentPropsType } from "types/Table";
-import { TableSeatedUser, User } from "types/User";
+import { Table } from "types/Table";
+import { TableSeatedUser } from "types/User";
 import { AnyVenue } from "types/venues";
 import { VenueTemplate } from "types/VenueTemplate";
 
 import { WithId } from "utils/id";
 import { generateTable } from "utils/table";
 import { arrayIncludes, isTruthy } from "utils/types";
 
+import { useAnalytics } from "hooks/useAnalytics";
 import { useExperience } from "hooks/useExperience";
 import { useSeatedTableUsers } from "hooks/useSeatedTableUsers";
 import { useShowHide } from "hooks/useShowHide";
+import { useUpdateTableRecentSeatedUsers } from "hooks/useUpdateRecentSeatedUsers";
+import { useUser } from "hooks/useUser";
 
 import { Loading } from "components/molecules/Loading";
 import { Modal } from "components/molecules/Modal";
 import { StartTable } from "components/molecules/StartTable";
 
 import { ButtonNG } from "components/atoms/ButtonNG";
 
-import "./TablesUserList.scss";
+import { TableComponent } from "../TableComponent";
 
-export interface TablesUserListProps {
-  setSeatedAtTable: (value: string) => void;
-  seatedAtTable: string | undefined;
+import styles from "./TableGrid.module.scss";
+
+interface TableGridProps {
   customTables: Table[];
   defaultTables: Table[];
   showOnlyAvailableTables?: boolean;
-  TableComponent: React.FC<TableComponentPropsType>;
   joinMessage: boolean;
   leaveText?: string;
   venue: WithId<AnyVenue>;
   venueId: string;
-  template: VenueTemplate;
-  user: WithId<User>;
+  userId: string;
 }
 
-export const TablesUserList: React.FC<TablesUserListProps> = ({
+export const TableGrid: React.FC<TableGridProps> = ({
```

Comment:
[Q] Should this component be moved into `./src/components/attendee/` or is it expected to be replaced by a freshly designed one?

[NOTE]
`TableGrid` as a name coined from two generic terms... 🤷 
Hopefully we will find simpler more specific sounding name in the future, like "huddle" is for communication

Reply:
Resolved in 8eea651b1